### PR TITLE
Shopify Base&Client configuration

### PR DESCRIPTION
**Pull request by Argonus**

### CODE REVIEW

**Comment by vfonic** [comment]
If possible, can you configure your editor to automatically add new line at the end of the file?

I have mine configured like that (and I think [many other people do](https://robots.thoughtbot.com/no-newline-at-end-of-file#following-the-rules-in-your-editor)), so if I edit your file, there will be some unnecessary changes at the end of the file.

There's really no much difference, so if you have strong preference, I can change my editor settings instead.

http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

**Comment by Argonus** [comment]
@vfonic Yep, it is temporary commit.

**Comment by vfonic** [comment]
👍

File: app/models/shopify_import/base.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+module ShopifyImport
+  class Base
+    def initialize(credentials: {}, client: nil)
+      @client = client(credentials) unless client
```
Comment (Mafi88) [inline app/models/shopify_import/base.rb]: Maybe 
```
@client = client || client(credentials)
```

**Pull request merged**